src/coreclr/src/vm/pgo.cpp

```diff
@@ -342,6 +483,166 @@ HRESULT PgoManager::getMethodBlockCounts(MethodDesc* pMD, unsigned ilSize, UINT3
     return E_NOTIMPL;
 }
 
+CORINFO_CLASS_HANDLE PgoManager::getLikelyClass(MethodDesc* pMD, unsigned ilSize, unsigned ilOffset, UINT32* pLikelihood, UINT32* pNumberOfClasses)
```

Comment:
Function header comment?

Reply:
I'll add one, but they're not as common in runtime code as they are in the jit.